src/pages/missions/MissionsHome.tsx

```diff
@@ -140,7 +140,7 @@ export default function MissionsHome({}: Props) {
           placeholder={"Search"}
           className={classes.searchInput}
           value={searchString}
-          onChange={(e) => setSearchString(e.target.value)}
+          onChange={(e) => setSearchString(e.target.value.trim())}
```

Comment:
Think this stops us from searching missions with a space in - looks like it works fine without this though!

```suggestion
          onChange={(e) => setSearchString(e.target.value))}
```